### ISSUE TITLE
Unexpected path behaviour from excluded file?

### ISSUE THREAD

**Issue by byCedric** [Microsoft/TypeScript-Sublime-Plugin#523]
**TL;DR;**
Opening a file from a `tsconfig.json` excluded directory in sublime doesn't resolve project files. But most of the other typescript features seems to be working... Is this expected behaviour? Or am I doing something terribly wrong? 

First of all, thanks for the wonderful TypeScript and the Sublime plugin! It has been a pleasure to work with. 

So I was setting up a simple project structure with TypeScript, Webpack, Mocha and React. The usual basics. When I created my first component everything went well and I got a fair build time; 2 seconds. Unfortunately after creating a (unit) test for this component, the build time soared to 20+ seconds... 

After some investigation I found out that TypeScript parsed all the testing dependencies, even though Webpack did not include the `/test` folder for the loader. But I think that is expected behaviour, since they stated that the `tsconfig.json`'s _exclude_ is used.

Because of this, I added the whole testing folder to the _exclude_ array. But after doing so, Sublime does not resolve the path to the component anymore. Instead it states `Cannot find module 'app/components/test'`. All other functionality seems to be working, autocompletion, typehinting, etc. Any input or even telling me I'm completely wrong is appreciated!

PS. All used files are available over [here](https://gist.github.com/byCedric/c4d6c29e545b74b81f7d2b85a7f328aa). Please note, I could not use directories so I used `-` as fake directory separator.
